Training/1.0 Introduction.md

```diff
@@ -12,37 +12,53 @@ First we we learn a little bit about coding in Python, although many of you will
 List all the Data Types of Python with examples.  
 
 ```
-
+list = ["one",2,"3"]
+string = "Hello World!"
+integer = 1024
+boolean = True
+dictionary = {"one":1,"two":2}
+tuple = (1,2,3,4)
 ```
 
 ### Task 2: 
 
 Other useful ways of storing data are in List, Arrays and Tuples. Create one of each and assign them to the variables a, b and c respectively. 
 
 ```
-
+a = ["I", "am", "a", "list"]
+b = [1,2,3,4]
```

Comment:
This is not how you make an array. The correct way is with numpy:
`import numpy
b = numpy.array([1,2,3,4])`